test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -404,6 +404,7 @@ public static TheoryData MultiAttributeCheckBoxData
                     tagHelper.For.ModelExplorer,
                     tagHelper.For.Name,
                     model,      // value
+                    null,       // format
```

Comment:
Please add a `format` parameter to this test as well as test cases where that value is non-`null`. Then use `format` here to confirm value is passed through to the generator as expected.

Also need a test in this class that confirms the `Format` property works correctly. Doesn't appear that's tested even when `type='text'` is chosen. Not your fault but since you're working in that area, it's a good time to fill that test gap.